projects/wave-core-new/src/lib/backend/backend.service.ts

```diff
@@ -73,8 +73,9 @@ export class BackendService {
             id: UUID,
             name?: string,
             description?: string,
-            layers?: Array<LayerDict>,
+            layers?: Array<LayerDict | 'none' | 'delete'>,
```

Comment:
What is the difference between none and delete?

Reply:
'none'  means no update and 'delete' means delete the layer